DirectProgramming/DPC++FPGA/ReferenceDesigns/merge_sort/src/static_math.hpp

```diff
@@ -1,5 +1,12 @@
-#ifndef __MISC_HPP__
-#define __MISC_HPP__
+#ifndef __STATICMATH_HPP__
+#define __STATICMATH_HPP__
+

```

Comment:
Should we put these functions in a namespace?  I'm thinking ahead in case we want to elevate this file to a common header library.

Reply:
Probably. What should the name of the namespace be? What if I rename the file common_utilities.hpp and make the namespace "common_util"??

Maybe sycl::fpga::common_util ?? Or something?

Reply:
Used namespace impu::math